ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -591,3 +592,29 @@ def get_all_storageclass():
         )
     ]
     return storageclass
+
+
+def setup_ceph_debug():
+    """
+    Set Ceph to run in debug log level using a ConfigMap.
+    This functionality is available starting OCS 4.7.
+
+    """
+    ceph_debug_log_configmap_data = templating.load_yaml(
+        constants.CEPH_CONFIG_DEBUG_LOG_LEVEL_CONFIGMAP
+    )
+    ceph_debug_log_configmap_data["data"]["config"] = (
+        "\n"
+        + "\n".join(constants.ROOK_CEPH_CONFIG_VALUES)
```

Comment:
When we resolve issue with `ROOK_CEPH_CONFIG_VALUES` representation (see comments above), this will become just `constants.ROOK_CEPH_CONFIG_VALUES + constants.CEPH_DEBUG_CONFIG_VALUES`.

Reply:
see my reply above

Reply:
fixed now